www/webapp/src/components/Field/RecordItem.vue

```diff
@@ -208,6 +202,31 @@ export default {
       return index < this.fields.length - 1
         && this.$refs.input[index].$refs.input.selectionStart === this.fields[index].value.length;
     },
+    keydownHandler(index, event) {
+      switch (event.keyCode) {
```

Comment:
The point of the change in Vue is that `[.keyCode` is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) (because it is implementation-dependent). Instead, it's now recommended to look for `.key` values: https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values

Reply:
Thank for the hint. My main intention was to get a (better) working vue3 test build.
Should I change it to `.key` in this PR or extra PR?

Reply:
Let's do it in this one :) -- If you have some time, can you send me an email please? Thanks.